documentation/modules/snip_vmware_esxi_nfc.adoc

```diff
@@ -0,0 +1,6 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+If you are using an EXSi host, you can configure the `vSphere Backup NFC` on the VMware site instead of specifying a host in your {project-short} migration plan.
```

Comment:
```suggestion
If you are using an ESXi host, you can configure `vSphere Backup NFC` on the VMware side instead of specifying a host in your {project-short} migration plan.
```
the changes above address phrasing/typo thingies but I also think we can drop the `If you are using an ESXi host` since ESXi hosts are always involved in migrations from vSphere? (or @RichardHoch , maybe I misunderstood what's the purpose of this part)
and also `instead of specifying a host in our MTV migration plan` - the transfer network on the VMware side can only be set at the host level, not at the plan level, no? (in that case, we need to drop it from documentation/modules/creating-migration-plan-2-6-3.adoc)